src/map/script.c

```diff
@@ -15203,21 +15203,27 @@ BUILDIN(npctalk)
 }
 
 // change npc walkspeed [Valaris]
-BUILDIN(npcspeed) {
-	struct npc_data* nd;
+static BUILDIN(npcspeed)
+{
+	struct npc_data *nd;
```

Comment:
nd and speed vars better use in same line with defining.
For example for ``nd``:
```struct npc_data *nd = map->id2nd(st->oid);```